### DISCUSSION TITLE
Using world downloader to dl 1.18 realm map.

### DISCUSSION THREAD

**Discussion by WhynotFulp** [mircokroon/minecraft-world-downloader#321] [Q&A]
Hello,

I found your mod on git hub and could use some advice. I have invested a lot of time into a realm and have become very attached to the map, the owner is for now still paying for it but lost access to admin privileges including the ability to download the map due to some kind of problem with migrating his account. He has not stopped paying for the realm but he is too lazy to contact Mojang customer support and get it sorted out, I am worried at some point he will cancel the subscription and I will lose the map. I have known about mc world downloader for quite some time and I was happy to see that you have kept it going for current Minecraft versions. I got help from another friend who is more technically savvy myself and he downloaded your mod and attempted to download the realm map for me but he ran into issues with the mod needing to grab the IP address of the realm and that not seemingly being possible. If I get him in contact with you over this email or via discord do you have experience in downloading maps from realms using your mod? the biggest problem seems to be the fact that realms changes its IP address often and it's difficult to obtain that IP address. Any advice is welcome, thank you.

-CM

**Comment by mircokroon** [comment]
There's a "Realms" tab in the GUI where you can pick the one you want to download from the list, you should be able to use this to get the connection details

**Reply by WhynotFulp** [reply]
![error](https://user-images.githubusercontent.com/99362800/153678192-185553c0-dcee-4966-8184-f4307be3e942.png)

This message has come up a few times. Any thoughts on what I need to do?

**Reply by WhynotFulp** [reply]
![local host 1](https://user-images.githubusercontent.com/99362800/153739145-d3eeece3-1158-4049-8326-1dfd73de8984.png)
![local host 2](https://user-images.githubusercontent.com/99362800/153739146-a5e582df-1d5f-4f41-a39f-942c4f11a828.png)
Your mod asks me to connect to local host, so far that is not working, any tips?

**Comment by WhynotFulp** [comment]
Thanks for getting back to me, may I add you on discord or email you in case he has any questions for you about downloading realms maps?
